docs/dev/docs.md

```diff
@@ -1,63 +1,46 @@
-
-
-Next, install font awesome for icons to display in the documentation.   
-`pip install fontawesome-markdown`
-
-and pymdown-extensions for tasklists:
-`pip install pymdown-extensions`
-
-
-## Setting up the MkDocs project
-
-:fa-info-circle: _**Important:**_ This step is only needed when setting up the project for documentation. Please ignore if you have already mkdocs.yml in the project directory.
-
-Go to the directory for the MkDocs project.   
-
-Then    
-
-`mkdocs new .`
+## Naming and Organization
 
-This command creates mkdocs.yml that will be used for MkDocs configuration, and docs/index.md, which is the Markdown file that is the entry point for the documentation.
+:fa-info-circle: _**Where are the documentation files?**_ ShapeWorks documentation is written using [Markdown](../dev/markdown.md#markdown-basics), a text layout language that enables simple formatting for section headers, code samples, weblinks, and images, yet is still readable as plain text. All documentation markdown files are located in the `docs/` folder. Documentation settings are configured by using the `mkdocs.yml` configuration file in the ShapeWorks directory.
 
-## Naming and Organization
 
 :fa-info-circle: **Naming convention:** To maintain consistency, please use lower-case letters and dashes for your markdown files, e.g., `fixed-domain-ellipsoid.md`.  
 
 :fa-info-circle: **Docs organization:** ShapeWorks documentation is organized into main sections (e.g., "Getting Started", "Use Cases", ... etc), which map to subfolders in the `docs/` folder (e.g., "getting-started", "use-cases", ... etc) and images used in their markdown files are located in the corresponding subfolders in the `docs/img` folder.
 
-## Building Documentation
+
+## Locally Building Documentation
   
+To see local changes to documentation in realtime, `cd` to where source documentation (i.e., `mkdocs.yml`) is maintained and run the following command:
+
 `mkdocs serve`
 
 This command builds markdown files into HTML and starts a development server to browse the documentation. 
 
 Open up `http://127.0.0.1:8000/` in your favorite browser to see the documentation. Changes you make to the markdown files are automatically rebuilt.
 
+
 ## Deploying on GitHub Pages
 
 As ShapeWorks is hosted on [GitHub](https://github.com/SCIInstitute/ShapeWorks), we use GitHub Pages to host the documentation for ShapeWorks. 
 
-We use Project Pages sites for documentation deployment. The site files are deployed to the `gh-pages' branch within the ShapeWorks repository.
+We use Project Pages sites for documentation deployment. The site files are deployed to the `gh-pages` branch within the ShapeWorks repository.
 
 To generate static HTML files for the markdown files, [checkout](build.md#clone-source) the branch where source documentation (`mkdocs.yml`) is maintained and run the following command:
 
 `mkdocs gh-deploy`
```

Comment:
A small note on style: most docs use triple ticks for "block" code, and single ticks for inline code

This is an example of `inline code`.

```
This is an example of block code.
```

The latter looks neater imo if its just code not accompanied by a sentence, and its possible to do programming language specific syntax highlight using: ```python
(i'm not sure if mkdocs supports this however)
```python
import numpy as np
```

Reply:
Good point. Could you point me to where in the docs this is needed to be changed?

Reply:
https://github.com/SCIInstitute/ShapeWorks/blob/e1a28e610407ec6bf3ffe78b879f71f306dbdca9/docs/dev/docs.md

The lines that says "mkdocs serve" and "mkdocs gh-deploy"

Reply:
Done